libs/velo-external-db-core/src/web/error-middleware.ts

```diff
@@ -2,7 +2,7 @@ import { NextFunction, Response } from 'express'
 import { domainToSpiErrorTranslator } from './domain-to-spi-error-translator'
 
 export const errorMiddleware = (err: any, _req: any, res: Response, _next?: NextFunction) => {
-  if (process.env['NODE_ENV'] !== 'test') {
+  if (process.env['NODE_ENV'] === 'test') {
```

Comment:
?

Reply:
I encountered some issues with the CI tests, and I needed to examine the errors. However, I inadvertently left the error messages visible in the tests. Since errors should not be present in tests, I have reverted the tests to their original state.

Reply:
bottom line, if i understand correctly, it was always `===`, and you changed it some when and now you're changing it back?

Reply:
Nope, the opposite it correct, it always was !==, that means the errors should not appers in tests, because there are tests that test errors and we don't want to see them in the tests results.. 
Changed it in `manged-adapter` and I forgot to fix this before rebasing this PR with `manged-apdater` branch ..